localization/english/load_tips/load_tips_l_english.yml

```diff
@@ -0,0 +1,114 @@
+﻿l_english:
+ LOADING_TIP_0:1 "Thank you for playing Warcraft II: Guardians of Azeroth. You Rock!"
+ LOADING_TIP_1:1 "“The hope for future generations has always resided in mortal hands. And now that my task is done, I will take my place...amongst the legends of the past.”\n #bold —Medivh#!"
+ LOADING_TIP_2:1 "#T Did you know?#!\n\nNight Elves are descendants of Trolls."
+ LOADING_TIP_3:1 "#T Did you know?#!\n\nDuskwood was originally called Brightwood."
+ LOADING_TIP_4:1 "#T Did you know?#!\n\nMglrmglmglmgl!"
+ LOADING_TIP_5:1 "#T Did you know?#!\n\nOgrezonia is a tiny island filled with giant female ogres that are rumored to perform horrible rituals to men who happen upon their isle. A ship from Azeroth Express crashed on the island..."
+ LOADING_TIP_6:1 "#T Did you know?#!\n\nAnduin Wrynn's favorite Pandarian food is lotus root paste buns."
+ LOADING_TIP_7:1 "“Not a Banshee Queen... Not a Ranger-General... But always a Windrunner.”\n #bold —Sylvanas Windrunner#!"
+ LOADING_TIP_8:1 "#T Did you know?#!\n\nSylvanas hates forest trolls."
+ LOADING_TIP_9:0 "#T Did you know?#!\n\nGarrosh did nothing wrong."
+ LOADING_TIP_10:0 "#T Did you know?#!\n\nThe Sha, physical manifestations of negative emotions, were created from the essence of the Old God Y'Shaarj."
+ LOADING_TIP_11:0 "“Pandaria, her hills of gold\nIn dark and mournful times of old\nDid once a hopeless horror hold\nWhen from her sacred veil did spring\nWith storm and flash, a monstrous thing\nHis name Lei Shen, the Thunder King...”\n #bold —Lorewalker Cho#!"
+ LOADING_TIP_12:0 "Humans, night elves, dwarves, gnomes, draenei, and the savage worgen make up the illustrious Alliance. Proud and noble, courageous and wise, these races work together to preserve order in Azeroth. The Alliance is driven by honor and tradition. Its rulers are champions of justice, hope, knowledge, and faith.\n\nIn a time when chaos and uncertainty reign, the Alliance remains steadfast in its determination to bring light to the darkest corners of the world."
```

Comment:
LOADING_TIP_12:0 "Humans, Night Elves, Dwarves, Gnomes, Draenei, and the savage Worgen make up the illustrious Alliance. Proud and noble, courageous and wise, these races work together to preserve order in Azeroth. The Alliance is driven by honor and tradition. Its rulers are champions of justice, hope, knowledge and faith.\n\nIn a time when chaos and uncertainty reign, the Alliance remains steadfast in its determination to bring light to the darkest corners of the world."